app/controllers/settings/integration.js

```diff
@@ -131,9 +155,13 @@ export default Controller.extend({
         deleteWebhook() {
             return this.webhookToDelete.destroyRecord();
         }
-
     },
 
+    updateIntegration: task(function* () {
+        yield this.store.findRecord('integration', this.integration.id, {backgroundReload: true});
```

Comment:
@kevinansfield Decided to reload the integration from server as was not sure whats the best pattern in this case, or if we should somehow hook the return value from api key update directly to refresh integration 😬

Reply:
Best practice would be to use [`store.pushPayload(response)`](https://api.emberjs.com/ember-data/release/classes/Store/methods/push?anchor=pushPayload) after [performing the reset](https://github.com/TryGhost/Ghost-Admin/pull/1562/files#diff-933b75966ff6010eae08c1d11b78fc15R23). There are a few examples of this in the codebase 🙂